keyboards/planck/keymaps/mwpeterson/keymap.c

```diff
@@ -241,6 +252,10 @@ void plover_lookup(void) {
   unregister_code(PV_RG);
 }
 
+void matrix_init_user(void) {
```

Comment:
```suggestion
void eeconfig_init_user(void) {
```
Calling it in `eeconfig_init_user` lets you set the input mode when EEPROM is reset, not just when the keyboard starts.